lib/flame/path.rb

```diff
@@ -11,7 +11,7 @@ class Path
 		## @param parts [Array<String, Flame::Path>] parts of expected path
 		## @return [Flame::Path] path from parts
 		def self.merge(*parts)
-			parts.join('/').gsub(%r{\/{2,}}, '/')
+			parts.join('/').gsub(%r[/{2,}], '/')
```

Comment:
Why?

Reply:
because shielding not needed if you use `%r[]`

Reply:
```ruby
pry: main > '/foo//bar///'.gsub(%r[/{2,}], '/')
=> "/foo/bar/"
pry: main > '/foo//bar///'.gsub(%r{/{2,}}, '/')
=> "/foo/bar/"
```

???

Can you give proof, please?

Reply:
oh, i meant that `%r{\/}` same that `%r{/}`

Reply:
OK. So, why did you change the brackets?

Reply:
because `r%[/{2,}]` looks more understandable than `r%{/{2,}}`

Reply:
Ruby Style Guide [recommends `%r|...|`](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces) for cases with `{}` inside `Regexp`.

Reply:
Got it. Changed.